src/index.ts

```diff
@@ -121,6 +121,7 @@ export * from "./decorator/controllers";
 export * from "./decorator/decorators";
 export * from "./decorator/methods";
 export * from "./decorator/params";
+export * from "./error/http/HttpError";
```

Comment:
I think we can add export for all other errors too, to make use of them convenient too